src/main/java/org/sonarsource/sonarlint/ls/CommandManager.java

```diff
@@ -171,15 +175,32 @@ private void computeCodeActionsForSonarLintIssues(Diagnostic diagnostic, List<Ei
         newCodeAction.setCommand(new Command(fix.message(), SONARLINT_QUICK_FIX_APPLIED, List.of(ruleKey)));
         codeActions.add(Either.forRight(newCodeAction));
       });
+
+      if (!hasNoBinding) {
+        var serverIssueKey = ((DelegatingIssue) versionedIssue.getIssue()).getServerIssueKey();
```

Comment:
What will happen if user has binding but the issue exists only locally?

Reply:
That's a very good catch. I've added another condition to check that the issue at hand is an instance of Delegating issue [a7b9dce](https://github.com/SonarSource/sonarlint-language-server/pull/246/commits/a7b9dce760338ff89c0fff30f0dad22aaed30606)